tonic-build/Cargo.toml

```diff
@@ -4,15 +4,15 @@ categories = ["network-programming", "asynchronous"]
 description = """
 Codegen module of `tonic` gRPC implementation.
 """
-documentation = "https://docs.rs/tonic-build/0.7.2/tonic_build/"
+documentation = "https://docs.rs/tonic-build/0.8.2/tonic_build/"
 edition = "2018"
 homepage = "https://github.com/hyperium/tonic"
 keywords = ["rpc", "grpc", "async", "codegen", "protobuf"]
 license = "MIT"
 name = "tonic-build"
 readme = "README.md"
 repository = "https://github.com/hyperium/tonic"
-version = "0.8.0"
```

Comment:
Looks like I forgot to bump the last PR and did it manually on release so this is why it jumped two.